project1/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Crane Wife: Short Story by CJ Hauser_Project 1</title>
+    <link rel="stylesheet" href="styles.css">
+    <link
+        href="https://fonts.googleapis.com/css?family=Aleo:400,400i,700i|Courgette|Denk+One|Fauna+One|Glegoo:400,700|Montserrat:300,300i,400,600,600i|Petit+Formal+Script|Satisfy|Syncopate:400,700&display=swap&subset=latin-ext"
+        rel="stylesheet">
+</head>
+
+
+<body>
+
+    <main>
+
+
+        <!-- I'm not sure if I am saving my commits and the file properly-->
+        <header style="text-align: center">
+            <h1>The Crane Wife</h1>
+            <p class="anchor"><a href="https://cjhauser.com/about-us">By CJ Hauser</a></p>
+            <p class="date">July 16, 2019</p>
+        </header>
+
+
+        <section>
+            <p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the
+                whooping crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for
+                nylon hiking
+                pants that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly
+                at
+                home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+                doing
+                that
+                requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and
+                floppy hat
+                featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that
+                looks
+                like
+                this when something in my life has gone so terribly wrong.</p>
+
+            <p>Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+                New
+                York house with two willow trees I had bought with my fiancé.</p>
+
+            <p>Ten days later and I didn’t want to do anything I was supposed to do.</p>
+        </section>
+
+        <div>
+            &bull; &bull; &bull;
+        </div>
+
+        <blockquote>
+            <p>"When I looked at that mouse with her broom, I wondered which one of us was wrong
+                about who
+                <span><strong><em>I</em></strong></span> was."</p>
```

Comment:
You can lose the `span` here and instead add your styles to the `strong` or `em` element. Also, I think you should opt for only one of them here -- probably `em`
```suggestion
                <em>I</em> was."</p>
```

If you did this, in the CSS you'd then change your `span` styles to:
```css
blockquote em {
    color: navy;
    opacity: 0.5;
}
```

This way, your styles _only_ apply to the `em` elements that are _inside_ `blockquote`.